packages/node_modules/samples/browser-call-with-screenshare/index.html

```diff
@@ -44,16 +44,46 @@
       <main>
         <h1>Local Screensharing</h1>
         <p>This sample demonstrates sharing your local screen</p>
+
+        <form id="typeCredentials">
+          <fieldset>
+            <legend>Pick Method of Getting Access Token</legend>
+              <div>
+                <input type="radio" id="oauth" name="drone" onchange="handleRadio()" value="oauth" checked>
```

Comment:
Making it more explicit as to what this represents
```suggestion
                <input type="radio" id="oauth-input" name="drone" onchange="handleRadio()" value="oauth" checked>
```